dashboard/config/honeybadger.yml

```diff
@@ -8,3 +8,8 @@ breadcrumbs:
 request:
   filter_keys:
     - "warden.user.user.key"
+
+# report data to honeybadger on the test server. this will not happen in unit tests
+# or ci because we unset the api key in those situations (see test.yml.erb).
+test:
+  report_data: true
```

Comment:
syntax derived from these sources:
* enable reporting: https://docs.honeybadger.io/lib/ruby/getting-started/environments/#development-environments
* limit to test env: https://docs.honeybadger.io/lib/ruby/getting-started/tests-and-honeybadger/#configuring-the-test-backend